app/screens/reaction_list/reaction_header.js

```diff
@@ -40,6 +60,7 @@ export default class ReactionHeader extends PureComponent {
                 highlight={selected === reaction.name}
                 onPress={this.handleOnPress}
                 theme={theme}
+                onLayout={(e) => this.handleOnLayout(e, reaction)}
```

Comment:
please refactor this in a way that you don't have anonymous functions here, perhaps the onLayout function from `reaction_header_item` should include the `reaction` in the event or as a second argument as you have here.